src/functionsDeployHelper.js

```diff
@@ -208,50 +208,13 @@ function pollDeploys(operations, printSuccess, printFail, printTooManyOps, proje
     });
 }
 
-function getRuntimeName(runtime) {
-  if (runtime === "nodejs8") {
-    return "Node.js 8";
-  }
-  if (runtime === "nodejs6") {
-    return "Node.js 6";
-  }
-  return runtime;
-}
-
 function getDefaultRuntime() {
-  // TODO uncomment when Node.js v8 is the default.
-  /**
-function getDefaultRuntime(options)
-  var packageJsonPath = path.join(
-    options.config.path(options.config.get("functions.source")),
-    "package.json"
+  utils.logWarning(
+    clc.bold.red(
```

Comment:
Let's do yellow, since logWarning will automatically put a little yellow :warning: emoji

Reply:
I wanted to really grab attention since this will be a breaking change in the future, although I can see that red is used to indicate errors and so might confuse the user into thinking something is wrong (something *is* wrong, though, that they should fix). These are the two options, I'm fine going with the yellow, defer to you.

Red:
![image](https://user-images.githubusercontent.com/31747099/57008536-b1e71c80-6ba5-11e9-8c41-3b93d8ac6604.png)

Yellow:
![image](https://user-images.githubusercontent.com/31747099/57008581-068a9780-6ba6-11e9-9cef-d5a113532484.png)

Reply:
Let's go with yellow, but maybe add newlines around it, so:

logger.info()
utils.logWarning(...)
logger.info()

I also think maybe we should add in something like:

```
For example:

{
  {"engines": {"node": "6"}}
}
```

That way it's really clear what they need to do.

Reply:
Changed to this:
![image](https://user-images.githubusercontent.com/31747099/57028868-0ebdf380-6bf5-11e9-8436-c251b4682077.png)

Reply:
LGTM but how about a little less indentation :)

Reply:
lololol fair, fair.

Reply:
![image](https://user-images.githubusercontent.com/31747099/57032710-e38cd180-6bff-11e9-83ee-334d02c58fcf.png)